src/main/java/org/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,20 @@
 package org.bookstore.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
```

Comment:
this can be a record

if your Checkstyle complaints on records - upgrade the plugin to 3.3.0